### DISCUSSION TITLE
All working with some minor modifications to the instructions.

### DISCUSSION THREAD

**Discussion by N6WT** [n1kdo/n1mm_view#35] [General]
I followed along with the instructions and things went well after I figured out to do chmod +x rpi_install.sh to make the file executable.

The only other problem I had was with the command to start the collector. "$ ./collector.py" was not working (bash: $: command not found). Neither did "./collector.py". The only way to get it to work was "sudo ./collector.py". If I did not use sudo the collector would start but nothing would be read into the DB file. Not sure why. 

Looks great and looking forward to next year when I can use this for our club. 

Raspberry Pi 4 Model B
Debain Raspbain 10 buster

Thanks for doing this!

73
Kent
N6WT

**Comment by n1kdo** [comment]
try `chmod  +x  collector.py` -- it seems the execute bit is not saved in github, most likely because I committed those files from Windows.

**Comment by n1kdo** [comment]
oh! don't type the $ in the command!  That is shown in the doc because it is the shell prompt.
